index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://www.google.com">My Logo</a>
+      <nav>
+        <img src="img/Union.png" alt="wite bars">  
+      </nav> 
+    </div>
+  </header>
```

Comment:
- Your design looks pretty nice there is just a small issue you still need to fix in the navigation bar to make it look like the Figma design. please add  a bit to the space between the nav menu and the logo the look as shown in the Figma design :+1: 

## Your design 

![a67208cd-e425-4e8d-974c-69dff9c2c936](https://user-images.githubusercontent.com/95297251/235542705-44783792-358f-4907-960a-218146031f2a.jpg)


## Figma design 

![image](https://user-images.githubusercontent.com/95297251/235543315-5c6b985b-4979-4b30-bed6-562fe5eb7aab.png)